auth/src/main/res/values/strings.xml

```diff
@@ -75,6 +75,10 @@
     <string name="fui_progress_dialog_sending" translation_description="Progress dialog while password recovery email is being sent">Sending…</string>
     <string name="fui_error_email_does_not_exist" translation_description="Inline error when user attempt signin with invalid account">That email address doesn\'t match an existing account</string>
 
+    <!-- Other error messages -->
+    <string name="fui_error_unknown" translation_description="General error messages for an unknown failure.">An unknown error occurred.</string>
```

Comment:
Not sure if this is intentional or not, but we already have a very similar string:
```xml
<string name="fui_general_error" translation_description="Toast text flashing general error">An error has occurred.</string>
```

Reply:
True.  I will remove that one, I like it with 'unknown'